README.md

```diff
@@ -169,6 +171,34 @@ You can import word templates by adding `<documenttemplate>` elements.
 </templateconfig>
 ```
 
+### Attribute specific functionality
+
+#### Set auto-number seed values
+
+When deploying auto-numbers, seed values can be defined in the template for each entity. These are set post-deployment. Setting the `<autonumberseedvalue>` element determines that the column is an auto-number.
+
+```xml
+<templateconfig>
+    <tables>
+      <table name="account">
+        <columns>
+          <column name="new_accountautonumber" autonumberseedvalue="1"/>
+        </columns>
+      </table>
+      <table name="contact">
+        <columns>
+          <column name="new_contactautonumber" autonumberseedvalue="2000"/>
+        </columns>
+      </table>
+    </tables>
+</templateconfig>
+```
+**Important Note**: When you set a seed value, it will reset the next number in the sequence to the seed value. Unless the autonumber atribute has an alternate key, it will not be enforced as unique. This means you could accidentially reset the count and end up with duplicate numbers.
```

Comment:
It's not clear to me from reading this that the package template handles this as long as the seed value in the config file doesn't change - 

```suggestion
**Important Note**: When you set a seed value, it will reset the next number in the sequence to the seed value. Unless the autonumber column has an alternate key, it will not be enforced as unique. This means you could accidentally reset the count and end up with duplicate autonumber values
```

Reply:
Sorry, I've just read the paragraph below!

Reply:
just the typos then 🙂